packages/integrations/vercel/README.md

```diff
@@ -214,26 +221,26 @@ You can use Vercel middleware to intercept a request and redirect before sending
 
 1. Add a `middleware.js` file to the root of your project:
 
-    ```js
-    // middleware.js
-    export const config = {
-      // Only run the middleware on the admin route
-      matcher: '/admin',
-    };
-
-    export default function middleware(request) {
-      const url = new URL(request.url);
-      // You can retrieve IP location or cookies here.
-      if (url.pathname === "/admin") {
-        url.pathname = "/"
-      }
-      return Response.redirect(url);
-    }
-    ```
+   ```js
+   // middleware.js
+   export const config = {
+     // Only run the middleware on the admin route
+     matcher: '/admin',
+   };
+
+   export default function middleware(request) {
+     const url = new URL(request.url);
+     // You can retrieve IP location or cookies here.
+     if (url.pathname === '/admin') {
+       url.pathname = '/';
+     }
+     return Response.redirect(url);
+   }
+   ```
+
 1. While developing locally, you can run `vercel dev` to run middleware. In production, Vercel will handle this for you.
 
-> **Warning**
-> **Trying to rewrite?** Currently rewriting a request with middleware only works for static files.
+> **Warning** > **Trying to rewrite?** Currently rewriting a request with middleware only works for static files.
```

Comment:
This looks wrong? Should be across two lines. Didn’t happen to other places we use the `> **Warning**` or `> **Note**` syntax, nor sure what the issue here was.

Reply:
Good catch! Looks like a prettier bug when the second line starts with a bold word